core/src/main/mima-filters/2.0.x.backwards.excludes/event-tags-insert-optimized.excludes

```diff
@@ -0,0 +1,10 @@
+ProblemFilters.exclude[IncompatibleSignatureProblem]("org.apache.pekko.persistence.jdbc.journal.dao.JournalTables#EventTags.eventId")
```

Comment:
Could you add an Apache source file header? The standard header as opposed to the Pekko one that mentions Akka.